bears/java/JavaPMDBear.py

```diff
@@ -17,22 +17,14 @@ class JavaPMDBear:
     """
 
     LANGUAGES = {'Java'}
+    REQUIREMENTS = {MavenRequirement('net.sourceforge.pmd:pmd', '5.6.0'),
```

Comment:
Sorry my bad just overlooked this and thought it would merely be adding the requirement line. Will fix this shortly.